Vic2ToHoI4/Data_Files/configurables/country_mappings.txt

```diff
@@ -207,8 +208,10 @@ link = { vic = SLO hoi = SLV } # Slovenia
 #link = { vic = ? hoi = SOL } # Solomon Islands
 link = { vic = SOM hoi = SOM } # Somalia
 link = { vic = RUS hoi = SOV } # Russia
+#link = { vic = CST hoi = SPA } # Castile => Nationalist Spain 
+link = { vic = SPC hoi = SPB } # Carlist Spain 
+link = { vic = ARN hoi = SPC } # Aragon
 link = { vic = SPA hoi = SPR } # Spain
-link = { vic = SPC hoi = SPR } # Carlist Spain => Spain
```

Comment:
Why shouldn't we map Carlist Spain to Carlist Spain?

Reply:
We are? I just moved the mapping of it above regular Spain's mapping because of alphabetical order.

Reply:
My bad.
@idhrendur, feel free to merge